### ISSUE TITLE
romeo_dcm_bringup has never been released

### ISSUE THREAD

**Issue by mikaelarguedas** [ros-aldebaran/romeo_dcm_robot#2]
`romeo_dcm_bringup` has never been released while `pepper_dcm_bringup` and `nao_dcm_bringup` are released (cf http://repositories.ros.org/status_page/compare_indigo_jade_kinetic.html?q=dcm_ and http://repositories.ros.org/status_page/ros_indigo_default.html?q=dcm_). Is it on purpose?

**Comment by nlyubova** [comment]
Yes, because I have never tested it on a robot but I would like to do it one day
